README.md

```diff
@@ -2,16 +2,15 @@
 
 The Plotly database connector is a cross-platform desktop application that connects [Plotly 2.0](https://plot.ly/alpha/workspace) to your database.
 
-![](http://g.recordit.co/NHyqNZEj2i.gif)
-![](http://g.recordit.co/BAyBgt0iRb.gif)
+![](http://g.recordit.co/LqhQcEcwti.gif)
 
 Plotly 2.0 makes HTTP requests from the local web browser directly to this database connector app. This database connector runs as a server on localhost and forwards queries from the Plotly 2.0 web-application to the database that connect to. Requests are made client-side, so you don't need to open up the connector or your database to the world, you just need to be able to access it from the machine that is running this connector app.
 
 [Learn more in our online documentation](http://help.plot.ly/database-connectors/) or just give it a try in [Plotly 2.0](https://plot.ly/alpha/workspace).
 
 
 #### Contact
-- Chris - chris@plot.ly
```

Comment:
I am not sure if you want me to remove this, but I was told the plan is to have me handle more the load of this project to give you more time for other projects so it might help if I handle the inquiries.